File: JavaScript/QUESTÃO 2.js
let a = Number(prompt("Enter a number for A"))
let b = Number(prompt("Enter a number for B"))
let c = Number(prompt("Enter a number for C"))

const quadraticEquation = b**2 -4*a*c

if (quadraticEquation > 0) {
    console.log("There are two distinct roots.")
} else if (quadraticEquation === 0) {
    console.log("There are two equal roots.")
} else {
    console.log("There are two imaginary roots")
}

File: Python/QUESTÃO 2.py

a = int(input("Enter a number for A: "))
b = int(input("Enter a number for B: "))
c = int(input("Enter a number for C: "))
quadraticEquation = b**2 -4*a*c
if quadraticEquation > 0:
    print("There are two distinct roots.")
elif quadraticEquation == 0:
    print("There are two equal roots.")
else:
    print("There are two imaginary roots.")